html/styles.css

```diff
@@ -0,0 +1,477 @@
+  background-image: url('../images/header-image.jpg');
+}
+
+.page-header .header-logo {
+  height: 100px;
+  margin-left: 100px;
+  margin-top: 20px;
+  margin-bottom: 0;
+  background-image: url('../images/adoptadog-logo.svg');
+  background-repeat: no-repeat;
+}
+
+.page-header .checkout-balance {
+  height: 30px;
+  color: white;
+  position: absolute;
+  top: 20px;
+  right: 100px;
+}
+
+.page-header .checkout-balance .cart-icon {
+  width: 20px;
+  margin-right: 2px;
+}
+
+.page-header .checkout-balance .dollar {
+  font-weight: 600;
+  font-size: 8pt;
+}
+
+.page-header .main-title {
+  margin-left: 100px;
+  margin-top: 0;
+  font-family: sans-serif;
+  font-size: 14pt;
+  color: #81B841;
+}
+
+.page-header .button-checkout {
+  background-color: #81B841;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  font-size: 7pt;
+}
+
+.page-header .button-checkout {
+  position: absolute;
+  right: 95px;
+  top: 50px;
+  text-transform: capitalize;
+  text-decoration: none; 
+}
+
+.page-header .button-checkout:hover {
+  opacity: 0.9;
+}
+
+.topnav {
+  background-color: #e0e0e0;
+  padding: 20px 10%;
+  font-size: 10pt;
+  
+  font-weight: bold;
+  word-spacing: 5px;
+}
+
+.navigation-link {
+  text-decoration: none;
+  color: #134350;
+  font-weight: 800;
+}
+
+main {
+  width: 80%;
+  margin: auto;
+  padding-bottom: 50px;
+}
+
+.home-main {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.home-section {
+  flex-shrink: 2;
+}
+
+.blog-preview {
+  flex-basis: 450px;
+  display: none;
+  padding: 0 4rem;
+ 
+}
+
+
+@media only screen and (min-width: 1200px) {
+  .blog-preview {
+    display: block;
+  }
+}
+
+.blog-preview .title {
+  color: #134350;
+  font-size: 1.8rem;
+  text-transform: capitalize;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.blog-preview-snippet {
+  margin-bottom: 4rem;
+}
+
+.blog-preview-snippet .picture {
+  display: block;
+  width: 100%;
+}
+
+.blog-preview-snippet .subtitle {
+  color: #134350;
+  font-size: 1.2rem;
+  margin: 1rem 0 0 0;
+  text-transform: capitalize;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.blog-preview-snippet .summary {
+  font-size: 0.9rem;
+  line-height: 1.4rem;
+}
+
+.blog-preview-snippet .read-more {
+  color: #8181b6;
+  font-size: 0.9rem;
+  text-decoration: none;
+}
+
+.blog-preview-snippet .read-more:hover {
+  text-decoration: underline;
+}
+.blog-preview-snippet .read-more::after {
+  content: " >>";
+}
+
+.introduction {
+  display: flex;
+  align-items: flex-start;
+  margin: 20px 0;
+}
+
+.introduction .intro-image {
+  width: 35%;
+  margin-right: 40px;
+}
+
+.introduction .intro-text .title {
+  color: #134350;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+.introduction .intro-text .intro-paragraph {
+  font-size: 12pt;
+  color: #444;
+  line-height: 25px;
+}
+
+@media only screen and (max-width:500px) {
+  .introduction {
+    flex-direction: column;
+    align-items: flex-start;
+  
+  }
+
+  .introduction .intro-image {
```

Comment:
Right now, when adjusting my screen size I noticed the into-image becomes tiny under tablet size. Using width percentage takes in from it's parent. Would look at using a responsive size like rem

Reply:
This is how it appears for me. It looked normal, but I will look into it.

![homepg-snippet](https://user-images.githubusercontent.com/60082512/101847281-bb699200-3b07-11eb-8bf4-ba03c1f9c13f.PNG)